nodeJS/authentication/project_members_only.md

```diff
@@ -16,6 +16,6 @@ This will be a chance for you to use the authentication skills we learned in the
 7. Display all member messages on the home page, but only show the author and date of the messages to other club-members.
 8. Add an optional field to the user model called Admin and then add the ability to delete messages, but only allow users who have `admin == true` to see the delete-button and delete messages.  You'll need to add a way to actually mark a user as an 'admin' so either add another secret pass-code page, or just put an "is admin" checkbox on the sign-up form.
 9. By this point, anyone who comes to the site should be able to see a list of all messages, with the author's name hidden. Users should be able to sign-up and create messages, but ONLY users that are members should be able to see the author and date of each message. Finally, you should have an Admin user that is able to see everything and also has the ability to delete messages. Obviously this is a simple and silly little app, but the things you are practicing (creating and authenticating users and giving users different abilities and permissions) are things that will be _very_ useful to you!
-10.  When you're satisfied with your work, deploy your project to heroku and share it below!
+10.  When you're satisfied with your work, deploy your project to PaaS of your choice and share it below!
```

Comment:
Can we edit this to say `deploy your project to the PaaS of your choice` please.

Reply:
Done!